wizeline/remix-project-lab
# Feat/related projects

# What does this PR do?
Implements "Add/Remove" input for Related Projects
Adds an Edit page route
Adds **_Related Projects_** section


# Where should the reviewer start?
Check the new edit page file:
`app/routes/projects/edit/$editProjectId.tsx`


# How should this be manually tested?

1. Run the application locally. Go to a project's page and click on the edit button. 
2. On the Edit page change the related projects values and submit. 
3. Go back to the project's page and check the values were saved correctly for the _**Related Projects**_ section



# Any background context you want to provide?


# What are the relevant tickets?
#52 


# Screenshots
<img width="1429" alt="Captura de Pantalla 2022-11-03 a la(s) 1 28 05" src="https://user-images.githubusercontent.com/90728303/199667015-e20e4841-89db-48ca-a3aa-9a249538107b.png">
<img width="1466" alt="Captura de Pantalla 2022-11-03 a la(s) 1 28 13" src="https://user-images.githubusercontent.com/90728303/199667030-3d45e8b2-d0a5-4ecb-b8b3-effe728b7610.png">



# Questions
In related projects section, should the Chips components be changed to Cards, o another component? 
Could we use useFetcher for the whole edit project form?


# Checklist
